### ISSUE TITLE
Move from NcMultiselect to NcSelect

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/tables#286]
NcMultiselect has lots of problems in regards to accessibility so at some point all usages should be switched to the new NcSelect component.
